doc/installation.rst

```diff
@@ -202,18 +205,17 @@ by running the following commands:
   # for the 'System Software'
   sudo apt-get install g++ autoconf automake libtool pkg-config libboost-filesystem-dev libboost-system-dev libyaml-cpp-dev
   # for the 'Python Software'
-  sudo apt-get install python3-dev libboost-python-dev python3-matplotlib python3-networkx python3-scipy python3-yaml
+  sudo apt-get install python3-dev libboost-python-dev python3-matplotlib python3-networkx python3-scipy python3-yaml python3-pip
   # for the 'Scientific Software'
   sudo apt-get install libgsl0-dev
 
-We recommend that you install ``dynesty``, ``pypmc`` and ``wilson``, and upgrade ``matplotlib`` to the latest available version, by running the following command:
+We strongly recommend the use of a python virtual environment (see :ref:`this FAQ answer <faq-virtualenv>` for more information about virtual environments) to install
+the ``dynesty``, ``matplotlib``, ``networkx``, ``scipy``, ``pypmc``, ``pyyaml``, and ``wilson`` packages:
 
-::
 
-  # for the 'pip3' command
-  apt-get install python3-pip
-  pip3 install --user dynesty pypmc wilson
-  pip3 install --user --upgrade matplotlib
+::
+  # Make sure to activate your virtual environment first
+  pip install dynesty===2.0.3 matplotlib networkx scipy pypmc pyyaml wilson
```

Comment:
Three equal signs?

Reply:
Good catch

Reply:
I am not 100% sure if this is correct or not. I think `===` has a specific meaning. To be checked :)

Reply:
It does have some specific meaning (see https://packaging.python.org/en/latest/specifications/version-specifiers/#version-matching) but it's not what we want